### ISSUE TITLE
Groupfolder quota greater 2 GB: Server replies Insufficent Storage.

### ISSUE THREAD

**Issue by RailsViceRoads** [nextcloud/groupfolders#98]
After applying [a320c7f ](https://github.com/nextcloud/groupfolders/commit/a320c7f170404600e65a342dce98ce39f0543072)as recommended in #93 the quota for some group foloders can now be set greater than 2 GB. But although on the folders is sufficient storage space, the server sends "Insufficient Storage (Sabre\DAV\Exception\InsufficentStorage)" to the NC Sync client.

Server
* NC 13.0.0.14
* Group folders 1.2.0

Client
* NC Client 2.3.3.1
* Windows 10

Group Folder 
* Used 3 GB, Quota 10 GB

Behaviour: Folder was synced down to client without errors. After Some files was modified by user, the "Not-Synced"-list of client is containing all modified files with the error message "Insufficient Storage (Sabre\DAV\Exception\InsufficentStorage)". All other group folder with quotas less then 2 GB works as expected.

I have set the quota to "unlimited" now. I'll write a comment to this issue if sync now works for the user.

![grafik](https://user-images.githubusercontent.com/36618192/37140559-6c86d6b8-22b2-11e8-8824-4b09275fd262.png)

**Comment by RailsViceRoads** [comment]
After I had set the quota to "unlimited" all clints synced her data without any additional user action. Looks like qota calculation can not handle quotas greater then 2 GB.

**Comment by fwmarcel** [comment]
Same Error

**Comment by kroerig** [comment]
Same error.

NC 13.0.2
Group Folders 1.2.0

Fileupload through browser.

**Comment by huste** [comment]
I can see the same wth 14.0.4 (I used 14.0.3 before where it started happening). I have set my users and group folder to 50GB; admittedly, I increased the grouop folder quota to 50 only a few days ago. I was getting the same errors as RailViceRoads posted in his screenshot. Plus on another client which also tried uploading files with another non-admin client gave me "Das Hochladen überschreitet das Speicherkontingent" (roughly: uploading fails due to exceeded quota). And that's after setting quotas to unlimited for the users and the shared folder and closing and restarting the client. Pressing "retry uploads" seems to now upload at least most of the files, while still nagging about others ("due to previous errors", as it states).
[...]
After restarting again, all files went into the cloud.

NextCloud 14.0.4
Group Folders 1.3.3

**Comment by icewind1991** [comment]
Do you have a 32bit php version?

**Comment by huste** [comment]
Must be 32bit, since echo PHP_INT_SIZE returns 4. Is there a more elgant way to check which php Nextcloud is actually using?

**Comment by ZeikoFr** [comment]
Hi @RailsViceRoads is the issue still relevant ?

**Comment by RailsViceRoads** [comment]
> Hi @RailsViceRoads is the issue still relevant ?

Today I have still all large group folders set to "unlimited". I'll check if the issue still exist next days.

**Comment by ZeikoFr** [comment]
and > > Hi @RailsViceRoads is the issue still relevant ?
> 
> Today I have still all large group folders set to "unlimited". I'll check if the issue still exist next days.

Thanks, I tried with my NC 19 and NC 20 instance and I could not replicate this issue on my end, if you encounter the issue please post more information about your actual environnement

**Comment by RailsViceRoads** [comment]
I have set all large group folders from "unlimited" to 8 or 16 GB at sunday. Till now I received no error reports from my users. I'll give you a final report end of the week.

By the way. Since I opened the issue I changed a lot:
* NC 20.0.2
* host changed from 32 to 64 bit system
* actual PHP version is now 7.4

Since host change, an other issue (https://github.com/nextcloud/server/issues/13160) is vanished without a trace. Could it be, that the integer representation has changed by change the system?
